### ISSUE TITLE
Delete droplets by tag missing

### ISSUE THREAD

**Issue by aakilfernandes** [lukeed/digitalocean-v2#7]
https://developers.digitalocean.com/documentation/v2/#deleting-droplets-by-tag

Looks like there's no method to this endpoint

**Comment by lukeed** [comment]
Ah, yup. This was released before there was any tagging functionality on DO.

Looks like you can still do this in the meantime:

```js
const DO2 = require('digitalocean-v2');
const API = new DO2({ token });

API.request(`droplets?tag_name=${tag}`, { method:'DELETE' }).then(...)
```

**Comment by aakilfernandes** [comment]
Sounds good. Should I close?

**Comment by lukeed** [comment]
Nah, it's okay. This will serve as a reminder for when I eventually get around to adding all methods.

Thanks!
